pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -396,7 +401,42 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			ObjectMeta: kapi.ObjectMeta{
 				Name: HPAControllerRoleName,
 			},
-			Rules: []authorizationapi.PolicyRule{},
+			Rules: []authorizationapi.PolicyRule{
+				// HPA Controller
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("get", "list"),
+					Resources: sets.NewString("horizontalpodautoscalers"),
+				},
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("horizontalpodautoscalers/status"),
+				},
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("get", "update"),
+					Resources: sets.NewString("replicationcontrollers/scale"),
+				},
+				{
+					Verbs:     sets.NewString("get", "update"),
+					Resources: sets.NewString("deploymentconfigs/scale"),
+				},
+				{
+					Verbs:     sets.NewString("create", "update", "patch"),
+					Resources: sets.NewString("events"),
+				},
+				// Heapster MetricsClient
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("pods"),
+				},
+				{
+					Verbs:         sets.NewString("proxy"),
+					Resources:     sets.NewString("services/api"),
```

Comment:
Opened https://github.com/kubernetes/kubernetes/pull/16570 to fix the URL parsing so you won't need to specify the `/api` subresource... please cherry pick that commit into this PR and fix the resource in this policy